app/layout/aside_feed.phtml

```diff
@@ -94,7 +94,7 @@
 						<a class="dropdown-toggle" data-fweb="<?= $feed->website() ?>"><?= _i('configure') ?></a>
 						<?php /* feed_config_template */ ?>
 					</div>
-					<?php if (FreshRSS_Context::$user_conf->show_favicons): ?><img class="favicon" src="<?= $feed->favicon() ?>" alt="✇" /><?php endif; ?>
+					<?php if (FreshRSS_Context::$user_conf->show_favicons): ?><img class="favicon" src="<?= $feed->favicon() ?>" alt="✇" loading="lazy" /><?php endif; ?>
```

Comment:
Does it actually make a difference here given how it doesn't depend on display:none? I ask mainly because the articles themselves are already lazy loaded.

Reply:
In the mobile view, this part is now shown, for instance. And in the main view, when categories are collapsed, it might work as well (not tested yet)